### PR TITLE
Update Dependabot configuration

### PR DESCRIPTION
**Pull request by gigitux**
Follow-up of #6836. If added, the labels option override the default configuration, so I added more labels.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ updates:
       open-pull-requests-limit: 10
       labels:
       - "skip-changelog"
+      - "type: dependencies"
+      - "github_actions"
```
Comment (dinhtungdu) [inline .github/dependabot.yml]:
```suggestion
          - 'skip-changelog'
          - 'type: dependencies'
          - 'github_actions'
```

Comment (dinhtungdu) [reply .github/dependabot.yml]: I think we should fix the indentation here to improve readability.

**Review by dinhtungdu — COMMENTED** [review]
I only left a suggestion to fix the indentation. Otherwise, this is LGTM!

**Comment by gigitux** [comment]
Thanks for the review! Now, it should be fine!

**Review by dinhtungdu — APPROVED** [review]
LGTM 🚀

**Pull request merged**